wordpress-mobile/WordPress-iOS
# Notifications: Fallback to Manual Height Calculation

### Details:
In the aims of fixing the "Invisible Cells" glitch we've been seeing (+ getting reports of), this PR restores Manual Height Calculation for Notifications.

I've been unable to pinpoint the exact issue that caused this bug (plus it's *very* hard to repro!). Since Autosizing has been buggy since it's initial release, let's fall back to the old and reliable code.

### To test:
1. Launch the app on iPhone / iPad Devices
2. Verify that the Notifications List looks as expected
3. Try Multitasking Modes. Verify nothing breaks

Sir, may i bug you with a code review?
Needs review: @kwonye
